aws_lambda_powertools/tracing/tracer.py

```diff
@@ -714,11 +714,11 @@ def _is_tracer_disabled() -> Union[bool, str]:
 
     def __build_config(
         self,
-        service: str = None,
-        disabled: bool = None,
-        auto_patch: bool = None,
-        patch_modules: Union[List, Tuple] = None,
-        provider: BaseProvider = None,
+        service: Optional[str] = None,
+        disabled: Optional[bool] = None,
+        auto_patch: Optional[bool] = None,
+        patch_modules: Union[List, Tuple, None] = None,
```

Comment:
is this correct? Shouldn't that be Optional[Sequence]?

Reply:
@heitorlessa  i will try that. There are a number of other places like this.

Reply:
mypy did not like this, so i change it to be Optional  like the others.